db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -13,9 +13,13 @@ def cloud?(prov_type)
 end
 
 def calculate_requested(options_hash = {})
-  {:memory  => get_total_requested(options_hash, :vm_memory) * 1024**2,
+  storage = get_total_requested(options_hash, :storage)
+  if %w(google).include?(@miq_request.source.vendor)
+    storage = get_option_value(@miq_request, :boot_disk_size).gigabytes
```

Comment:
@billfitzgerald0120 
Is this for a single vm, should this get multipled by the the number of VM's that we are provisioning. All other calls seems to be going thru get_total_requested.
Also can we make this into an if/else block.

if %w(google).include?(@miq_request.source.vendor)
  storage = ...
else
   storage = get_total_requested(options_hash, :storage)
end